_posts/2016-07-01-essay-theme.markdown

```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+title: essay-theme
+date: 2016-07-01 19:00
+homepage: https://github.com/mnichangxin/essay/
+download: https://github.com/mnichangxin/essay/archive/master.zip
+demo: http://mnichangxin.com/essay/
+author: Changxin Li
+thumbnail: essay.jpg
+license: MIT License
+license_link: https://github.com/Adagio-cantabile/Adagio-cantabile.github.io/blob/master/LICENSE.md
```

Comment:
I think that this should be `https://github.com/mnichangxin/essay/blob/master/LICENSE.md`.